app/helpers/application_helper.rb

```diff
@@ -307,6 +307,15 @@ def name_title(args)
     end
   end
 
+  def link_to_search_value(args)
+    if args[:document][args[:field]].present?
+      args[:document][args[:field]].each_with_index do |field_t, i|
```

Comment:
the variable name `field_t` is misleading. `field` is consistent with the other methods